package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@eppo/js-client-sdk-common",
-  "version": "1.8.1",
+  "version": "1.8.2",
```

Comment:
In theory there is a breaking API change here, as `useNonExpiringAssignmentCache` is getting renamed, practically only our SDKs are calling this directly. If you'd like to bump this to 1.9 or more we can discuss.

Reply:
It seems like that's a major version change, then, as it's not backward compatible. If we bump the minor version or patch number, our customers may pull this in on publish when they install/upgrade. To prevent that, I think we'll need to go `2.0.1`